rotkehlchen/tests/utils/ethereum.py

```diff
@@ -277,9 +278,11 @@ def setup_ethereum_transactions_test(
         ],
     )
 
-    if one_receipt_in_db:
+    if one_receipt_in_db is True:
         with database.user_write() as cursor:
             dbethtx.add_receipt_data(cursor, txreceipt_to_data(expected_receipt1))
+            if second_receipt_in_db is True:
```

Comment:
Why this change here?

Reply:
because the previous implementation didn't add the second receipt in order to query it from etherscan and test that functionality. For this test I don't need to query etherscan as it is only checking that the decoding works as expected and wanted to reuse this logic that we already had